### PR TITLE
Code Insights: Add backend insight story card

### PR DESCRIPTION
**Pull request by vovakulikov**
Closes part of https://github.com/sourcegraph/sourcegraph/issues/23189

### Context
This PR does preparation work for the drill-down filters panel at the backend insight card

- Change the gql method API in order to be able to mock the fetch method in storybook stories
- Adds the backend card storybook story 
- Fix global styles usage (now all global styles must be included via one single insight styles file at the top level of /insights dir)

### CODE REVIEW

File: client/web/src/insights/pages/dashboards/dashboard-page/components/dashboards-content/components/empty-insight-dashboard/EmptyInsightDashboard.tsx [hunk 1]
```diff
@@ -76,7 +76,7 @@ export const EmptySettingsBasedDashboard: React.FunctionComponent<EmptyInsightDa
                 </div>
             </button>
             <span className="d-flex justify-content-center mt-3">
-                <Link to="/insights/create">...or create a new insight</Link>
+                <Link to="/insights/create">...or create new insight</Link>
```
Comment (eseliger) [inline client/web/src/insights/pages/dashboards/dashboard-page/components/dashboards-content/components/empty-insight-dashboard/EmptyInsightDashboard.tsx:79]: Wasn't this correct?

**Review by eseliger — APPROVED** [review]

**Review by tjkandala — APPROVED** [review]

File: client/web/src/insights/pages/dashboards/dashboard-page/components/dashboards-content/components/empty-insight-dashboard/EmptyInsightDashboard.tsx (hunk 1 above)
Comment (vovakulikov) [reply client/web/src/insights/pages/dashboards/dashboard-page/components/dashboards-content/components/empty-insight-dashboard/EmptyInsightDashboard.tsx:79]: @AlicjaSuska Just checking are we sure that we want to stick with `or create new insight` copy here?

**Comment by vovakulikov** [comment]
@AlicjaSuska I'm about to merge this PR let's back to the all code insight-related copy in a separate PR.

**Comment by AlicjaSuska** [comment]
Thank you @vovakulikov

I've discussed the 'a' /no 'a' problems with @quinnkeast:

> “create a new insight” makes it more conversational, but that conversational vibe isn’t needed here

I agree with this statement. Thank you for making the change :)

**Pull request merged**